routes/webhook.js

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2017, The Linux Foundation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+const express = require('express');
+const yaml = require('js-yaml');
+
+const router = express.Router();
+const validPrActions = ['opened', 'reopened', 'synchronize'];
+let webhookSecret = null;
+let curationService;
+
+router.post('/', async (request, response, next) => {
+  const isGithubEvent = request.headers['x-github-event'];
+  if (!isGithubEvent) {
+    return fatal(request, response, 'Not a Github event');
+  }
+
+  // @todo secure webhook, see https://github.com/Microsoft/ghcrawler/blob/develop/routes/webhook.js#L28
+
+  const {pull_request: pr, action: prAction, number} = request.body;
+  const {sha, ref} = pr.head;
+  const isValidPullRequest = pr && validPrActions.includes(prAction);
+  if (!isValidPullRequest) {
+    return fatal(request, response, 'Not a valid Pull Request event');
+  }
+
+  const prFiles = await curationService.getPrFiles({number});
+  const curationFilenames = prFiles.map(
+    x => x.filename
+  ).filter(curationService.isCurationFile);
+
+  const curationResults = await Promise.all(
+    curationFilenames.map(
+      path => curationService.getContent({ref, path})
+        .then(curationService.isValidCuration)
+    )
+  );
+  const invalidCurations = [];
+  curationResults.forEach((result, index) => {
+    if (!result) {
```

Comment:
FWIW, I have been NOT using {} for single like expressions like this. I'm open to group consensus but lets be consistent until a change is decided on

Reply:
Yep, w/e everyone agrees to. I personally always use `{}`, but we should shoot for consistency 👍